ros_ws/src/car_control/include/dms_controller.h

```diff
@@ -5,17 +5,16 @@
 #include <std_msgs/Int64.h>
 #include <std_msgs/String.h>
 
-// How often the dead mans switch is checked. in Hz
-constexpr const int DMS_CHECK_RATE = 20;
-
-// How old the last dead mans switch check can be. in ms
-constexpr const int DMS_EXPIRATION = 100;
+constexpr const char* DMS_CHECK_RATE_PARAMETER = "/dms_controller/dms_check_rate";
```

Comment:
**MAY**
Add the Prefix "TOPIC_" to emphasis both are topics.

Reply:
These are not topics, but parameter names.

Reply:
In that case it is kind of weird that we (as far as I can see in the code base) use "TOPIC_" as a prefix for a topic **but** "_PARAMETER" as a suffix for a parameter.